app.js

```diff
@@ -8,7 +8,8 @@ const Material = require('./app/renderer/materialize');
 const components = {
     "task-suite": require('./app/renderer/components/task_suite'),
     "navbar": require('./app/renderer/components/navbar'),
-    "about": require('./app/renderer/components/about')
+    "about": require('./app/renderer/components/about'),
+    "bottombar": require('./app/renderer/components/bottombar'),
```

Comment:
By convention in gaucho/css, componentes names should be separate using `-`, in this case, `bottombar` should be `bottom-bar`

Reply:
The file name should be renamed to `bottom_bar.js` as well please